schematics/scully/src/add-post/index.ts

```diff
@@ -7,7 +7,7 @@ import {Schema} from './schema';
 export default function(options: Schema): Rule {
   return (host: Tree, context: SchematicContext) => {
 
-    const name = options.title ? options.title : 'blog-X';
+    const name = options.name ? options.name : 'blog-X';
     if (!host.exists(`./blog/${name}.md`)) {
```

Comment:
I don't think that we should hardcode the path `./blog` should we? @jorgeucano did you add the option for people to call the folder something besides `blog`? if so, we need to refactor this so that this doesn't have the path hardcoded.